dot/rpc/modules/childstate.go

```diff
@@ -29,6 +29,13 @@ type GetKeysRequest struct {
 	Hash   common.Hash
 }
 
+// GetChildStorageRequest the request to get the entry child storage hash
+type GetChildStorageRequest struct {
+	KeyChild []byte
+	EntryKey []byte
+	Hash     common.Hash
```

Comment:
can this be a `*common.Hash` instead as the block hash is optional?

Reply:
Done!